libenkf/src/model_config.c

```diff
@@ -546,18 +510,30 @@ void model_config_init(model_config_type * model_config ,
   else
     model_config->has_prediction = false;
 
-
-  if (config_content_has_item(config ,  CASE_TABLE_KEY))
-    model_config_set_case_table( model_config , ens_size , config_content_iget( config , CASE_TABLE_KEY , 0,0));
-
   if (config_content_has_item( config , ENSPATH_KEY))
     model_config_set_enspath( model_config , config_content_get_value_as_path(config , ENSPATH_KEY));
 
   if (config_content_has_item( config , DATA_ROOT_KEY))
     model_config_set_data_root( model_config , config_content_get_value_as_path(config , DATA_ROOT_KEY));
 
-  if (config_content_has_item( config , JOBNAME_KEY))
+  /*
```

Comment:
Am I understanding it correctly if we could now require `JOBNAME` to be present and then have a boolean flag `ECL` that would toogle the settings in `ecl_config`? I understand that we have legacy, config files that should still work and so forth, just wondering to check that I got it correctly.

Also, will the docs be updated accordingly?